decidim-core/config/locales/en.yml

```diff
@@ -1159,16 +1159,16 @@ en:
         main_image: Main image
     newsletters:
       unsubscribe:
-        check_subscription: If you want to change your preferences, you can do so in the <a href="%{link}" target="_blank">configuration page</a>
-        error: There was a problem unsubscribing
+        check_subscription: If you want to change your preferences, you can do so in the <a href="%{link}" target="_blank">configuration page</a>.
+        error: There was a problem unsubscribing.
         success: You are unsubscribed successfully.
         token_error: The link has expired.
         unsubscribe: Unsubscribe
     newsletters_opt_in:
-      unathorized: Sorry, this link is no longer available
+      unathorized: Sorry, this link is no longer available.
       update:
-        error: Something wrong happened
-        success: Newsletter settings successfully updated
+        error: Something wrong happened.
```

Comment:
I think we should favor the "There was a problem ..." formula, as that's what we use more: 

```bash
$ grep -r "There was a problem" decidim*/config/locales/en.yml |wc -l 
275
``` 

```suggestion
        error: There was a problem saving the newsletter settings.
```